sklearn/lda.py

```diff
@@ -180,6 +180,12 @@ class LDA(BaseEstimator, LinearClassifierMixin, TransformerMixin):
     covariance_ : array-like, shape (n_features, n_features)
         Covariance matrix (shared by all classes).
 
+    explained_variance_ratio_ : array, [n_components]
```

Comment:
just replace the line with what I suggest. The formatting is not consistent with the rest of the docstring.